extensions/blocks/contact-form/edit.js

```diff
@@ -319,8 +321,11 @@ function JetpackContactFormEdit( {
 
 			<InspectorControls>
 				<PanelBody title={ __( 'Form Settings', 'jetpack' ) }>{ renderFormSettings() }</PanelBody>
-				<PanelBody title={ __( 'CRM Integration', 'jetpack' ) } initialOpen={ false }>
+				{ ! isSimpleSite() && (
 					<CRMConnectionSettings jetpackCRM={ jetpackCRM } setAttributes={ setAttributes } />
+				) }
+				<PanelBody title={ __( 'Newsletter Integration', 'jetpack' ) } initialOpen={ false }>
+					<NewsletterIntegrationSettings />
```

Comment:
Could you wrap this into a `isSimpleSite` check, like we do just above? This is necessary because the Jetpack plugin is used on WordPress.com, but we cannot show that panel there since folks cannot install plugins.